apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/(analysis)/summary/components/share-survey-modal.test.tsx

```
@@ -1,348 +1,314 @@
-  SESSION_MAX_AGE: 1000,
-  REDIS_URL: "test-redis-url",
-  AUDIT_LOG_ENABLED: true,
-  IS_FORMBRICKS_CLOUD: false,
+// Mock child components to simplify testing
+vi.mock("./shareEmbedModal/share-view", () => ({
+  ShareView: ({ tabs, activeId, setActiveId, survey }: any) => (
+    <div data-testid="share-view">
+      <div data-testid="survey-type">{survey.type}</div>
+      <div data-testid="active-tab">{activeId}</div>
+      {tabs.map((tab: any) => (
+        <button key={tab.id} data-testid={`tab-${tab.id}`} onClick={() => setActiveId(tab.id)}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)